src/tests/Microsoft.PowerFx.Core.Tests/TexlTests.cs

```diff
@@ -3859,6 +3859,35 @@ public void TestSilentValidDelegatableFilterPredicateNode(string script, bool wa
             {
                 symbol.RemoveFunction(silentFilterFunction);
             }
+        }
+
+        [Theory]
+        [InlineData("T.A", "*[A:n]", true)]
+        [InlineData("T.B", "*[B:![C:s]]", true)]
+        [InlineData("T.A", "e", false)]
+        [InlineData("T.B", "e", false)]
+        public void TestSignleColumnAccessDependentOnTableType(string script, string expectedSchema, bool isBindingSuccess)

```

Comment:
>isBindingSuccess [](http://example.com/codeflow?start=106&length=16)

Instead of specifying here what we expect, consider calling this parameter 'usePowerFxV1Rules' or something similar, to make it clearer. And in the validation you can check that if PFxV1, then error, otherwise success.